kuma/static/js/wiki.js

```diff
@@ -976,10 +976,40 @@
         initDetailsTags();
     }
 
-    // clear out any drafts from localStorage, now that the document has been saved
-    if (typeof document_saved !== 'undefined' && document_saved) {
-      localStorage.removeItem('draft/edit' + location.pathname);
-      localStorage.removeItem('draft/edit' + location.pathname + '#save-time');
+
+    //
+    // Generates a storage key based on pathname
+    // copied from wiki-edit-draft.js because: race conditions
+    // does not need to copy logic for dealing with new translations
+    //
```

Comment:
We should probably start moving to some cross project standards for comments. In this case we probably want to do:

```
/**
 * Generates a storage key based on pathname
 * copied from wiki-edit-draft.js because: race conditions
 * does not need to copy logic for dealing with new translations
 */
```
Probably a good idea to follow the JSDoc guidelines http://usejsdoc.org/ - unless MDN already uses another JSDoc standard or tool.